contracts/GringottsBank.sol

```diff
@@ -129,6 +131,30 @@ contract  GringottsBank is DSAuth, BankSettingIds {
         }
     }
 
+     function burndrop(uint256 _depositID, address _depositor, uint48 _months, uint48 _startAt, uint64 _unitInterest, uint128 _value, bytes _data) public {
+         // Check the validity of the deposit
+         require(deposits[_depositID].claimed == false, "Already claimed");
+         require(deposits[_depositID].startAt > 0, "Deposit not created.");
+         require(deposits[_depositID].depositor == msg.sender, "Permission denied");
+
+         // check params
+         require(deposits[_depositID].depositor == _depositor, "Params error: _depositor");
+         require(deposits[_depositID].months == _months, "Params error: months");
+         require(deposits[_depositID].startAt == _startAt, "Params error: startAt");
+         require(deposits[_depositID].unitInterest == _unitInterest, "Params error: unitInterest");
+         require(deposits[_depositID].value == _value, "Params error: amount");
+
+         require(_data.length == 32, "The address (Darwinia Network) must be in a 64-bit hexadecimal format");
+
+         // TODO status
+         deposits[_depositID].claimed = true;
+
+         removeUserDepositsByID(_depositID, msg.sender);
```

Comment:
Only remove user's deposit ID is not enough, the relative amount of RING that the bank are holding require to be burned from the token balances.

Reply:
```suggestion
        address ring = registry.addressOf(SettingIds.CONTRACT_RING_ERC20_TOKEN);
        IBurnableERC20(ring).burn(address(this), _value);
```